tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "aegir/src/config/tsconfig.aegir.json",
   "compilerOptions": {
-    "outDir": "dist"
+    "outDir": "dist",
+    "lib": ["ES2021", "ES2021.Promise", "ES2021.String", "ES2020.BigInt", "DOM", "DOM.Iterable", "WebWorker"],
```

Comment:
Is it worth adding the `"WebWorker"` entry to the lib list in aegir instead?

Reply:
yea we should definitely add it in aegir. let me do that

Reply:
i'm slightly worried about making the aegir lib list too long, which could make some users think things are available that aren't, since it's just removing typing errors, we could lose the benefit of strict typings.

 aegir does test all the environments though, so it's probably fine?

Reply:
opened https://github.com/ipfs/aegir/pull/1219

Reply:
> i'm slightly worried about making the aegir lib list too long

https://github.com/ipfs/aegir/pull/1196/files updates the target/module to be more modern which lets us make the lib list shorter?